doc/Command-Reference.md

```diff
@@ -105,6 +105,9 @@
 * [Troubleshooting Commands](#troubleshooting-commands)
 * [Routing Stack](#routing-stack)
 * [Quagga BGP Show Commands](#Quagga-BGP-Show-Commands)
+* [Container Auto-restart](#container-autorestart-commands)
+  * [Container Auto-restart show commands](#container-autorestart-show-commands)
+  * [Container Auto-restart config command](#container-autorestart-config-command)
```

Comment:
Please move this section so that it is in alphabetical order (between BGP and DHCP relay).

Reply:
Fixed.